test/refactoring/ConvertAsync/convert_to_continuation.swift

```diff
@@ -0,0 +1,351 @@
+// RUN: %empty-directory(%t)
+
+func withAsyncAlternative(completionHandler: (Int) -> Void) {}
+func withAsyncAlternative() async -> Int {}
+func withAsyncThrowingAlternative(completionHandler: (Int?, Error?) -> Void) {}
+func withAsyncThrowingAlternative() async throws -> Int {}
+
+func withoutAsyncAlternativeBecauseOfMismatchedCompletionHandlerName(closure: (Int) -> Void) {}
+func withoutAsyncAlternativeBecauseOfReturnValue(completionHandler: (Int) -> Void) -> Bool {}
+func withoutAsyncAlternativeThrowing(closure: (Int?, Error?) -> Void) {}
+func asyncVoidWithoutAlternative(completionHandler2: () -> Void) {}
+
```

Comment:
Need some tests for a completion handler with a `Result` argument and then also handling to use `continuation(with: result)` as well

Reply:
Good catch, the entire `Result` case wasn’t covered by the implementation yet. I added two test cases now.